alf/examples/ddpg_conf.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2022 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# default ddpg config
```

Comment:
The ``ddpg_pendulum.gin``  in repo also need to be converted?

Reply:
Good point, this is also caught by unit test, which is good.  Done.

Also updated gin and README for Haonan's comments:
> Also need to convert ddpg_fetchslide.gin because it depends on ddpg_fetchreach.gin, and update the readme link after the conversion.